app/Model/Feed.php

```diff
@@ -485,7 +485,8 @@ public function attachFeedCorrelations(array $attributes, array $user, array &$e
             }
 
             if (in_array($attribute['type'], $compositeTypes, true)) {
-                list($value1, $value2) = explode('|', $attribute['value']);
+                $parts = explode('|', $attribute['value']) ?? [null, null];
```

Comment:
When this problem occurs?

Reply:
indeed this error was due to improper attribute values that were ingested directly in the DB, should never happen